genie-core/src/main/java/com/netflix/genie/core/properties/ExponentialBackOffProperties.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.core.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.Min;
+
+/**
+ * Properties to configure an ExponentialBackOff.
+ *
+ * @author mprimi
+ * @since 3.3.8
+ */
+
+@Getter
+@Setter
+@Validated
+public class ExponentialBackOffProperties {
```

Comment:
You're planning this to be an `@ConfigurationProperties`?

Reply:
It's not standalone, it's a sub-tree of JobsProperties, as such I didn't think that's necessary (it was not on the other properties classes this sits next to)